1209/6804/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Hootswitch
+owner: saybur
+license: GPLv3 (firmware), CERN-OHL-S-2.0 (hardware)
+site: https://github.com/saybur/hootswitch
+source: https://github.com/saybur/hootswitch
+---
```

Comment:
Thanks for reviewing, description added.

[https://pid.codes/howto/](https://pid.codes/howto/) has this noted as optional, no problem adding it but it may be worth clarifying in the guide.